src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -53,12 +79,16 @@ const FolderViewBody = ({
       const isNote = models.file.isNote(file)
       const isShortcut = models.file.isShortcut(file)
 
-      //Should only be called if mobile
       if (isShortcut) {
-        const resp = await client.query(
-          Q('io.cozy.files.shortcuts').getById(file.id)
-        )
-        window.location.href = resp.data.attributes.url
+        if (isMobileApp()) {
+          const resp = await client.query(
+            Q('io.cozy.files.shortcuts').getById(file.id)
+          )
+          window.location.href = resp.data.attributes.url
+        } else {
+          const url = generateFileUrl({ file, client, isFlatDomain: true })
```

Comment:
true on flatdomain ? 🤔 We have method now to get the right value, right?

Reply:
Yep, it's corrected in a later commit.